app/views/shared/_recipe.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="bg-white rounded shadow p-4 flex">
+  <!-- Left Column -->
+  <div class="w-1/4 pr-4">
+    <h2 class="text-xl font-bold">
+      <%= link_to recipe.title, recipe_path(recipe), class: 'hover:underline' %>
+    </h2>
```

Comment:
![Screenshot (151)](https://github.com/Donmark2k/food_recipe/assets/87555548/f51e0910-e80e-4101-a28d-0e21a46179ec)
when you change from ` link_to recipe.title ` to `link_to recipe.name ` it works fine